### DISCUSSION TITLE
one security code should be attached ?

### DISCUSSION THREAD

**Discussion by Nazrina412** [learndevopsonline/learndevopsonline#1938] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/4ba3721f-e07b-4db1-9d8f-140cb1c11566)

**Comment by chaithanya1812** [comment, marked as answer]
### It's ["allow-all"]
![image](https://github.com/user-attachments/assets/f4de8522-e020-4eb4-8e65-7bcc39e00967)
